Makefile

```
@@ -18,8 +18,8 @@ export EXTRA_VARS ?= "@$(5GC_ROOT_DIR)/vars/main.yml"
 		--extra-vars "ROOT_DIR=$(ROOT_DIR)" --extra-vars $(EXTRA_VARS)
 
 #### b. Provision k8s ####
-5gc-install: 5gc-router-install 5gc-core-install
-5gc-uninstall: 5gc-core-uninstall 5gc-router-uninstall
+5gc-install: 5gc-router-install 5gc-sriov-install 5gc-core-install
+5gc-uninstall: 5gc-core-uninstall 5gc-sriov-uninstall 5gc-router-uninstall
```

I think this needs to be reverted back to how it was before, and you need to create two different targets (specific for SRIOV deployment(s) as shown below). Otherwise, you will be always deploy/execute the `5gc-sriov-(un)install`
```suggestion
5gc-install: 5gc-router-install 5gc-core-install
5gc-uninstall: 5gc-core-uninstall 5gc-router-uninstall
5gc-some_name-install: 5gc-router-install 5gc-sriov-install 5gc-core-install
5gc-some_name-uninstall: 5gc-core-uninstall 5gc-sriov-uninstall 5gc-router-uninstall
```

---

the sriov tasks will run based on the mode we configure. For af_packet mode these tasks will be ignore. So lets keep this target as it is and it will be good to maintain same target irrespective of deployment mode.